spec/controllers/organization_alliances_controller_spec.rb

```diff
@@ -0,0 +1,155 @@
+RSpec.describe OrganizationAlliancesController do
+  let(:organization) { Fabricate(:organization) }
+  let(:other_organization) { Fabricate(:organization) }
+  let(:member) { Fabricate(:member, organization: organization, manager: true) }
+  let(:user) { member.user }
+
+  before do
+    login(user)
+  end
+
+  describe "GET #index" do
+    let!(:pending_sent) {
+      OrganizationAlliance.create!(
+        source_organization: organization,
+        target_organization: other_organization,
+        status: "pending"
+      )
+    }
+
+    let!(:pending_received) {
+      OrganizationAlliance.create!(
+        source_organization: Fabricate(:organization),
+        target_organization: organization,
+        status: "pending"
+      )
+    }
+
+    let!(:accepted) {
+      OrganizationAlliance.create!(
+        source_organization: organization,
+        target_organization: Fabricate(:organization),
+        status: "accepted"
+      )
+    }
+
+    let!(:rejected) {
+      OrganizationAlliance.create!(
+        source_organization: organization,
+        target_organization: Fabricate(:organization),
+        status: "rejected"
+      )
+    }
```

Comment:
use this let at the beggining, create a fabricator for this model and if the ! after let is not needed don't use it